docs/usage/self-hosted-configuration.md

```diff
@@ -98,6 +98,14 @@ Set to `false` to prevent usage of `--ignore-platform-reqs` in the Composer pack
 
 This configuration will be applied after all other environment variables so that it can be used to override defaults.
 
+## dockerChildPrefix
+
+Adds a custom prefix to the default renovate sidecar Docker containers name and label.
+
+If this is set to `myprefix_` the final image name for `renovate/node` would be named `myprefix_renovate_node` instead of currently used `renovate_node` and be labeled `myprefix_renovate_child` instead of `renovate_child`.
```

Comment:
@rarkins What about changing all manager to only pass `node` instead of `renovate/node` this would solve the naming issue. 

And we can simply join the `dockerImagePrefix`.

Reply:
Sure

Reply:
@Chumper what do you think about this?

Reply:
I am not sure if this would solve anything.

The name of the container is derived from the registry + image, in this case `renovate/node`.
Passing the only `node` to manager would solve what problem?
The naming of the container would still be `renovate_node`; if the default is `renovate_` we would still end up with `renovate_renovate_node`.

Can you give some examples?

e.g. 
image: `renovate/node`
prefix: `myprefix_`
=> 
container_name: 
container_label

Reply:
e.g.
dockerImagePrefix: ghcr.io/renovatebot
image: node
prefix: myprefix_
=>
container_name: myprefix_node
container_label: myprefix_child
container_image: ghcr.io/renovatebot/node

Reply:
ok, that would change the calculation of the container name.
Before we derived it from registry + image as pointed above, now it will just be prefix + image.

If you are fine with that than I can apply those changes.

Reply:
yes, that why i proposed to only pass the image name without registry and repository from the managers.

so they pass `image=node` instead of `image=renovate/node`

Reply:
This change is still pending, right?

Reply:
yes, i didn't have the time yet to make the changes. I will update the branch once I did.